frontend/smart-kickers-game/src/apis/Game.spec.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { resetGame } from './Game';
+
+describe('resetGame API caller', () => {
+  let mock;
+
+  beforeAll(() => {
+    mock = new MockAdapter(axios);
+  });
+
+  afterEach(() => {
+    mock.reset();
+  });
+
+  it('should send reset game request', async () => {
+    mock.onPut(`http://localhost:3000/reset`).reply(200);
+
+    const result = await resetGame();
+
+    expect(result.status).toBe(200);
+  });
+
+  it('should ignore backend errors (result not updated)', async () => {
+    mock.onPut(`http://localhost:3000/reset`).reply(500);
+
+    const result = await resetGame();
+
+    expect(result.status).toBe(500);
+  });
```

Comment:
Would you kindly you point me to the code area this case is testing?

Reply:
Actually, it's checking that resetGame returns the correct status code

Reply:
But isn't that already checked in the case before?

Reply:
I get that we want to check if for X `reply` that the `status` would be X (not always 200), but if that's needed let's use Jest's `each` feature.

Reply:
It's no case (in my opinion) with error handling)